frontend/packages/dev-console/integration-tests/features/web-terminal/web-Terminal-adminuser.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Web Terminal for Developer user
+    As a user with admin rights, I should be able to use web terminal
+
+
+Background:
+    Given user has logged in as admin user
+    And user is at developer perspecitve
+    And user has installed Web Terminal operator
+    And user has created Web Terminal instance
+    And user has created namespace "aut-terminal-admin"
+    And user is at the Topolgy page
+
+
+@regression, @smoke
+Scenario: Create new project and use Web Terminal
+    Given user is at topology page
+    When user clicks on the Web Terminal icon on the Masthead
+    And user selects Create Project from Project drop down menu
+    And user enters project name "aut-terminal-adminuser"
+    Then user will see the terminal window for namespace "aut-terminal-adminuser"
```

Comment:
Is there normally a step in between here that requires the user to submit the form?

Reply:
It's there. I will add a new statement where user submits the form.